06-Jun-2025 05:28:15.504 SEVERE [Catalina-utility-1] org.apache.catalina.core.StandardContext.listenerStart Exception sending context initialized event to listener instance of class [org.artifactory.lifecycle.webapp.servlet.ArtifactoryHomeConfigListener]
	java.lang.IllegalStateException: Could not find database table: db_properties
		at org.artifactory.converter.VersionProviderImpl.init(VersionProviderImpl.java:150)
		at org.artifactory.lifecycle.webapp.servlet.BasicConfigurationManager.initVersion(BasicConfigurationManager.java:173)
		at org.artifactory.lifecycle.webapp.servlet.BasicConfigurationManager.initArtifactoryInstallation(BasicConfigurationManager.java:150)
		at org.artifactory.lifecycle.webapp.servlet.BasicConfigurationManager.initialize(BasicConfigurationManager.java:137)
		at org.artifactory.lifecycle.webapp.servlet.ArtifactoryHomeConfigListener.initBasicConfigManager(ArtifactoryHomeConfigListener.java:61)
		at org.artifactory.lifecycle.webapp.servlet.ArtifactoryHomeConfigListener.contextInitialized(ArtifactoryHomeConfigListener.java:53)
		at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4046)
		at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:4474)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:164)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:599)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:571)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:654)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:632)
		at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1880)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
		at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
	Caused by: java.lang.IllegalStateException: Could not find database table: db_properties
		at org.artifactory.common.config.db.DbVersionDao.isDbPropertiesTableExists(DbVersionDao.java:51)
		at org.artifactory.converter.VersionProviderImpl.tryToResolveFromDb(VersionProviderImpl.java:238)
		at org.artifactory.converter.VersionProviderImpl.getCompoundVersionDetailsFromDbIfExists(VersionProviderImpl.java:163)
		at org.artifactory.converter.VersionProviderImpl.loadOriginalDbVersion(VersionProviderImpl.java:184)
		at org.artifactory.converter.VersionProviderImpl.init(VersionProviderImpl.java:146)
		... 20 more
	Caused by: org.postgresql.util.PSQLException: The connection attempt failed.
		at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:364)
		at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:54)
		at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:273)
		at org.postgresql.Driver.makeConnection(Driver.java:446)
		at org.postgresql.Driver.connect(Driver.java:298)
		at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:137)
		at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:360)
		at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:202)
		at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:461)
		at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:550)
		at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:98)
		at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:111)
		at org.jfrog.storage.wrapper.JFrogDataSourceWrapper.getConnection(JFrogDataSourceWrapper.java:104)
		at org.jfrog.storage.wrapper.JFrogDataSourceWrapper.invoke(JFrogDataSourceWrapper.java:73)
		at jdk.proxy2/jdk.proxy2.$Proxy5.getConnection(Unknown Source)
		at org.jfrog.storage.util.DbUtils.withMetadata(DbUtils.java:765)
		at org.jfrog.storage.util.DbUtils.tableExists(DbUtils.java:275)
		at org.artifactory.common.config.db.DbVersionUtil.isDbPropertiesTableExists(DbVersionUtil.java:50)
		at org.artifactory.common.config.db.DbVersionDao.isDbPropertiesTableExists(DbVersionDao.java:48)
		... 24 more
	Caused by: java.net.SocketTimeoutException: Connect timed out
		at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:546)
		at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592)
		at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
		at java.base/java.net.Socket.connect(Socket.java:751)
		at org.postgresql.core.PGStream.createSocket(PGStream.java:260)
		at org.postgresql.core.PGStream.<init>(PGStream.java:121)
		at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:140)
		at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:268)
		... 42 more
06-Jun-2025 05:28:15.518 SEVERE [Catalina-utility-1] org.apache.catalina.core.StandardContext.listenerStart Exception sending context initialized event to listener instance of class [org.artifactory.lifecycle.webapp.servlet.ArtifactoryContextConfigListener]
	java.lang.IllegalStateException: Artifactory home not initialized.
		at org.artifactory.lifecycle.webapp.servlet.ArtifactoryContextConfigListener.getArtifactoryHome(ArtifactoryContextConfigListener.java:208)
		at org.artifactory.lifecycle.webapp.servlet.ArtifactoryContextConfigListener.setSessionTrackingMode(ArtifactoryContextConfigListener.java:189)
		at org.artifactory.lifecycle.webapp.servlet.ArtifactoryContextConfigListener.init(ArtifactoryContextConfigListener.java:102)
		at org.artifactory.lifecycle.webapp.servlet.ArtifactoryContextConfigListener.contextInitialized(ArtifactoryContextConfigListener.java:90)
		at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4046)
		at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:4474)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:164)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:599)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:571)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:654)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:632)
		at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1880)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
		at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 05:28:15.615 SEVERE [Catalina-utility-1] org.apache.catalina.core.StandardContext.listenerStop Exception sending context destroyed event to listener instance of class [org.artifactory.lifecycle.webapp.servlet.LogbackConfigListener]
	java.lang.NullPointerException: Cannot invoke "ch.qos.logback.classic.selector.ContextSelector.detachLoggerContext(String)" because "selector" is null
		at org.jfrog.common.logging.logback.servlet.LogbackConfigManager.destroy(LogbackConfigManager.java:108)
		at org.artifactory.lifecycle.webapp.servlet.LogbackConfigListener.contextDestroyed(LogbackConfigListener.java:115)
		at org.apache.catalina.core.StandardContext.listenerStop(StandardContext.java:4092)
		at org.apache.catalina.core.StandardContext.stopInternal(StandardContext.java:4700)
		at org.apache.catalina.util.LifecycleBase.stop(LifecycleBase.java:235)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:168)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:599)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:571)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:654)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:632)
		at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1880)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
		at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 05:38:17.654 SEVERE [Catalina-utility-1] org.apache.catalina.core.StandardContext.listenerStart Exception sending context initialized event to listener instance of class [org.artifactory.lifecycle.webapp.servlet.ArtifactoryHomeConfigListener]
	java.lang.IllegalStateException: Could not find database table: db_properties
		at org.artifactory.converter.VersionProviderImpl.init(VersionProviderImpl.java:150)
		at org.artifactory.lifecycle.webapp.servlet.BasicConfigurationManager.initVersion(BasicConfigurationManager.java:173)
		at org.artifactory.lifecycle.webapp.servlet.BasicConfigurationManager.initArtifactoryInstallation(BasicConfigurationManager.java:150)
		at org.artifactory.lifecycle.webapp.servlet.BasicConfigurationManager.initialize(BasicConfigurationManager.java:137)
		at org.artifactory.lifecycle.webapp.servlet.ArtifactoryHomeConfigListener.initBasicConfigManager(ArtifactoryHomeConfigListener.java:61)
		at org.artifactory.lifecycle.webapp.servlet.ArtifactoryHomeConfigListener.contextInitialized(ArtifactoryHomeConfigListener.java:53)
		at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4046)
		at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:4474)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:164)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:599)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:571)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:654)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:632)
		at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1880)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
		at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
	Caused by: java.lang.IllegalStateException: Could not find database table: db_properties
		at org.artifactory.common.config.db.DbVersionDao.isDbPropertiesTableExists(DbVersionDao.java:51)
		at org.artifactory.converter.VersionProviderImpl.tryToResolveFromDb(VersionProviderImpl.java:238)
		at org.artifactory.converter.VersionProviderImpl.getCompoundVersionDetailsFromDbIfExists(VersionProviderImpl.java:163)
		at org.artifactory.converter.VersionProviderImpl.loadOriginalDbVersion(VersionProviderImpl.java:184)
		at org.artifactory.converter.VersionProviderImpl.init(VersionProviderImpl.java:146)
		... 20 more
	Caused by: org.postgresql.util.PSQLException: The connection attempt failed.
		at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:364)
		at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:54)
		at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:273)
		at org.postgresql.Driver.makeConnection(Driver.java:446)
		at org.postgresql.Driver.connect(Driver.java:298)
		at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:137)
		at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:360)
		at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:202)
		at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:461)
		at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:550)
		at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:98)
		at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:111)
		at org.jfrog.storage.wrapper.JFrogDataSourceWrapper.getConnection(JFrogDataSourceWrapper.java:104)
		at org.jfrog.storage.wrapper.JFrogDataSourceWrapper.invoke(JFrogDataSourceWrapper.java:73)
		at jdk.proxy2/jdk.proxy2.$Proxy5.getConnection(Unknown Source)
		at org.jfrog.storage.util.DbUtils.withMetadata(DbUtils.java:765)
		at org.jfrog.storage.util.DbUtils.tableExists(DbUtils.java:275)
		at org.artifactory.common.config.db.DbVersionUtil.isDbPropertiesTableExists(DbVersionUtil.java:50)
		at org.artifactory.common.config.db.DbVersionDao.isDbPropertiesTableExists(DbVersionDao.java:48)
		... 24 more
	Caused by: java.net.SocketTimeoutException: Connect timed out
		at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:546)
		at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592)
		at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
		at java.base/java.net.Socket.connect(Socket.java:751)
		at org.postgresql.core.PGStream.createSocket(PGStream.java:260)
		at org.postgresql.core.PGStream.<init>(PGStream.java:121)
		at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:140)
		at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:268)
		... 42 more
06-Jun-2025 05:38:17.660 SEVERE [Catalina-utility-1] org.apache.catalina.core.StandardContext.listenerStart Exception sending context initialized event to listener instance of class [org.artifactory.lifecycle.webapp.servlet.ArtifactoryContextConfigListener]
	java.lang.IllegalStateException: Artifactory home not initialized.
		at org.artifactory.lifecycle.webapp.servlet.ArtifactoryContextConfigListener.getArtifactoryHome(ArtifactoryContextConfigListener.java:208)
		at org.artifactory.lifecycle.webapp.servlet.ArtifactoryContextConfigListener.setSessionTrackingMode(ArtifactoryContextConfigListener.java:189)
		at org.artifactory.lifecycle.webapp.servlet.ArtifactoryContextConfigListener.init(ArtifactoryContextConfigListener.java:102)
		at org.artifactory.lifecycle.webapp.servlet.ArtifactoryContextConfigListener.contextInitialized(ArtifactoryContextConfigListener.java:90)
		at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4046)
		at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:4474)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:164)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:599)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:571)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:654)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:632)
		at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1880)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
		at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 05:38:17.700 SEVERE [Catalina-utility-1] org.apache.catalina.core.StandardContext.listenerStop Exception sending context destroyed event to listener instance of class [org.artifactory.lifecycle.webapp.servlet.LogbackConfigListener]
	java.lang.NullPointerException: Cannot invoke "ch.qos.logback.classic.selector.ContextSelector.detachLoggerContext(String)" because "selector" is null
		at org.jfrog.common.logging.logback.servlet.LogbackConfigManager.destroy(LogbackConfigManager.java:108)
		at org.artifactory.lifecycle.webapp.servlet.LogbackConfigListener.contextDestroyed(LogbackConfigListener.java:115)
		at org.apache.catalina.core.StandardContext.listenerStop(StandardContext.java:4092)
		at org.apache.catalina.core.StandardContext.stopInternal(StandardContext.java:4700)
		at org.apache.catalina.util.LifecycleBase.stop(LifecycleBase.java:235)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:168)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:599)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:571)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:654)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:632)
		at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1880)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
		at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 11:47:05.265 SEVERE [http-nio-8081-exec-1] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [Jersey Web Application] in context with path [/artifactory] threw exception
	java.lang.NullPointerException: Cannot invoke "org.artifactory.api.context.ArtifactoryContext.beanForType(java.lang.Class)" because the return value of "org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryContext()" is null
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryTracer(ArtifactoryTracingFilter.java:89)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.startServerSpan(ArtifactoryTracingFilter.java:84)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.doFilter(ArtifactoryTracingFilter.java:38)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.valves.rewrite.RewriteValve.invoke(RewriteValve.java:300)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:116)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:398)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:903)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1740)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1189)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:658)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 11:47:11.956 SEVERE [Catalina-utility-2] org.apache.catalina.core.StandardContext.listenerStop Exception sending context destroyed event to listener instance of class [org.artifactory.lifecycle.webapp.servlet.LogbackConfigListener]
	java.lang.NullPointerException: Cannot invoke "ch.qos.logback.classic.selector.ContextSelector.detachLoggerContext(String)" because "selector" is null
		at org.jfrog.common.logging.logback.servlet.LogbackConfigManager.destroy(LogbackConfigManager.java:108)
		at org.artifactory.lifecycle.webapp.servlet.LogbackConfigListener.contextDestroyed(LogbackConfigListener.java:115)
		at org.apache.catalina.core.StandardContext.listenerStop(StandardContext.java:4092)
		at org.apache.catalina.core.StandardContext.stopInternal(StandardContext.java:4700)
		at org.apache.catalina.util.LifecycleBase.stop(LifecycleBase.java:235)
		at org.apache.catalina.core.ContainerBase$StopChild.call(ContainerBase.java:1219)
		at org.apache.catalina.core.ContainerBase$StopChild.call(ContainerBase.java:1208)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
		at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 11:49:35.856 SEVERE [http-nio-127.0.0.1-8091-exec-1] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [Jersey Web Application] in context with path [/artifactory] threw exception
	java.lang.NullPointerException: Cannot invoke "org.artifactory.api.context.ArtifactoryContext.beanForType(java.lang.Class)" because the return value of "org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryContext()" is null
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryTracer(ArtifactoryTracingFilter.java:89)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.startServerSpan(ArtifactoryTracingFilter.java:84)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.doFilter(ArtifactoryTracingFilter.java:38)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.valves.rewrite.RewriteValve.invoke(RewriteValve.java:300)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:116)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:398)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:903)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1740)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1189)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:658)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 11:49:36.242 SEVERE [http-nio-8081-exec-1] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [Jersey Web Application] in context with path [/artifactory] threw exception
	java.lang.NullPointerException: Cannot invoke "org.artifactory.api.context.ArtifactoryContext.beanForType(java.lang.Class)" because the return value of "org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryContext()" is null
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryTracer(ArtifactoryTracingFilter.java:89)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.startServerSpan(ArtifactoryTracingFilter.java:84)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.doFilter(ArtifactoryTracingFilter.java:38)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.valves.rewrite.RewriteValve.invoke(RewriteValve.java:300)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:116)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:398)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:903)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1740)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1189)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:658)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 11:49:37.389 SEVERE [http-nio-8081-exec-2] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [Jersey Web Application] in context with path [/artifactory] threw exception
	java.lang.NullPointerException: Cannot invoke "org.artifactory.api.context.ArtifactoryContext.beanForType(java.lang.Class)" because the return value of "org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryContext()" is null
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryTracer(ArtifactoryTracingFilter.java:89)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.startServerSpan(ArtifactoryTracingFilter.java:84)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.doFilter(ArtifactoryTracingFilter.java:38)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.valves.rewrite.RewriteValve.invoke(RewriteValve.java:300)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:116)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:398)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:903)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1740)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1189)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:658)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 11:49:38.424 SEVERE [http-nio-8081-exec-3] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [Jersey Web Application] in context with path [/artifactory] threw exception
	java.lang.NullPointerException: Cannot invoke "org.artifactory.api.context.ArtifactoryContext.beanForType(java.lang.Class)" because the return value of "org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryContext()" is null
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryTracer(ArtifactoryTracingFilter.java:89)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.startServerSpan(ArtifactoryTracingFilter.java:84)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.doFilter(ArtifactoryTracingFilter.java:38)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.valves.rewrite.RewriteValve.invoke(RewriteValve.java:300)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:116)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:398)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:903)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1740)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1189)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:658)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 11:49:39.473 SEVERE [http-nio-8081-exec-4] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [Jersey Web Application] in context with path [/artifactory] threw exception
	java.lang.NullPointerException: Cannot invoke "org.artifactory.api.context.ArtifactoryContext.beanForType(java.lang.Class)" because the return value of "org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryContext()" is null
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryTracer(ArtifactoryTracingFilter.java:89)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.startServerSpan(ArtifactoryTracingFilter.java:84)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.doFilter(ArtifactoryTracingFilter.java:38)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.valves.rewrite.RewriteValve.invoke(RewriteValve.java:300)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:116)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:398)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:903)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1740)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1189)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:658)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 11:49:40.503 SEVERE [http-nio-8081-exec-5] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [Jersey Web Application] in context with path [/artifactory] threw exception
	java.lang.NullPointerException: Cannot invoke "org.artifactory.api.context.ArtifactoryContext.beanForType(java.lang.Class)" because the return value of "org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryContext()" is null
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryTracer(ArtifactoryTracingFilter.java:89)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.startServerSpan(ArtifactoryTracingFilter.java:84)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.doFilter(ArtifactoryTracingFilter.java:38)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.valves.rewrite.RewriteValve.invoke(RewriteValve.java:300)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:116)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:398)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:903)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1740)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1189)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:658)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 11:49:41.524 SEVERE [http-nio-8081-exec-7] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [Jersey Web Application] in context with path [/artifactory] threw exception
	java.lang.NullPointerException: Cannot invoke "org.artifactory.api.context.ArtifactoryContext.beanForType(java.lang.Class)" because the return value of "org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryContext()" is null
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryTracer(ArtifactoryTracingFilter.java:89)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.startServerSpan(ArtifactoryTracingFilter.java:84)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.doFilter(ArtifactoryTracingFilter.java:38)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.valves.rewrite.RewriteValve.invoke(RewriteValve.java:300)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:116)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:398)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:903)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1740)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1189)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:658)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 11:49:42.543 SEVERE [http-nio-8081-exec-6] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [Jersey Web Application] in context with path [/artifactory] threw exception
	java.lang.NullPointerException: Cannot invoke "org.artifactory.api.context.ArtifactoryContext.beanForType(java.lang.Class)" because the return value of "org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryContext()" is null
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.getArtifactoryTracer(ArtifactoryTracingFilter.java:89)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.startServerSpan(ArtifactoryTracingFilter.java:84)
		at org.artifactory.webapp.servlet.ArtifactoryTracingFilter.doFilter(ArtifactoryTracingFilter.java:38)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.valves.rewrite.RewriteValve.invoke(RewriteValve.java:300)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:116)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:398)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:903)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1740)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1189)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:658)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 12:24:39.864 INFO [Catalina-utility-1] org.apache.catalina.core.ApplicationContext.log Marking servlet [Binary Store] as unavailable
06-Jun-2025 12:24:39.866 SEVERE [Catalina-utility-1] org.apache.catalina.core.StandardContext.loadOnStartup Servlet [Binary Store] in web application [/artifactory] threw load() exception
	java.lang.ClassNotFoundException: org.jfrog.storage.binstore.servlet.BinaryStoreServlet
		at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1380)
		at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1189)
		at org.apache.catalina.core.DefaultInstanceManager.loadClass(DefaultInstanceManager.java:493)
		at org.apache.catalina.core.DefaultInstanceManager.loadClassMaybePrivileged(DefaultInstanceManager.java:475)
		at org.apache.catalina.core.DefaultInstanceManager.newInstance(DefaultInstanceManager.java:143)
		at org.apache.catalina.core.StandardWrapper.loadServlet(StandardWrapper.java:764)
		at org.apache.catalina.core.StandardWrapper.load(StandardWrapper.java:704)
		at org.apache.catalina.core.StandardContext.loadOnStartup(StandardContext.java:4221)
		at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:4508)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:164)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:599)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:571)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:654)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:632)
		at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1880)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
		at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 12:39:55.230 SEVERE [Catalina-utility-2] org.apache.catalina.core.StandardContext.listenerStop Exception sending context destroyed event to listener instance of class [org.artifactory.lifecycle.webapp.servlet.LogbackConfigListener]
	java.lang.NullPointerException: Cannot invoke "ch.qos.logback.classic.selector.ContextSelector.detachLoggerContext(String)" because "selector" is null
		at org.jfrog.common.logging.logback.servlet.LogbackConfigManager.destroy(LogbackConfigManager.java:108)
		at org.artifactory.lifecycle.webapp.servlet.LogbackConfigListener.contextDestroyed(LogbackConfigListener.java:115)
		at org.apache.catalina.core.StandardContext.listenerStop(StandardContext.java:4092)
		at org.apache.catalina.core.StandardContext.stopInternal(StandardContext.java:4700)
		at org.apache.catalina.util.LifecycleBase.stop(LifecycleBase.java:235)
		at org.apache.catalina.core.ContainerBase$StopChild.call(ContainerBase.java:1219)
		at org.apache.catalina.core.ContainerBase$StopChild.call(ContainerBase.java:1208)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
		at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
06-Jun-2025 13:06:36.269 INFO [Catalina-utility-1] org.apache.catalina.core.ApplicationContext.log Marking servlet [Binary Store] as unavailable
06-Jun-2025 13:06:36.274 SEVERE [Catalina-utility-1] org.apache.catalina.core.StandardContext.loadOnStartup Servlet [Binary Store] in web application [/artifactory] threw load() exception
	java.lang.ClassNotFoundException: org.jfrog.storage.binstore.servlet.BinaryStoreServlet
		at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1380)
		at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1189)
		at org.apache.catalina.core.DefaultInstanceManager.loadClass(DefaultInstanceManager.java:493)
		at org.apache.catalina.core.DefaultInstanceManager.loadClassMaybePrivileged(DefaultInstanceManager.java:475)
		at org.apache.catalina.core.DefaultInstanceManager.newInstance(DefaultInstanceManager.java:143)
		at org.apache.catalina.core.StandardWrapper.loadServlet(StandardWrapper.java:764)
		at org.apache.catalina.core.StandardWrapper.load(StandardWrapper.java:704)
		at org.apache.catalina.core.StandardContext.loadOnStartup(StandardContext.java:4221)
		at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:4508)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:164)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:599)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:571)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:654)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:632)
		at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1880)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
		at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
